src/detect-filemd5.c

```diff
@@ -78,57 +78,38 @@ static int DetectFileMd5Setup (DetectEngineCtx *de_ctx, Signature *s, const char
 static int MD5MatchLookupString(ROHashTable *hash, const char *string)
 {
     uint8_t md5[16];
-    if (ReadHashString(md5, string, "file", 88, 32) == 1) {
-        void *ptr = ROHashLookup(hash, &md5, (uint16_t)sizeof(md5));
-        if (ptr == NULL)
-            return 0;
-        else
-            return 1;
-    }
-    return 0;
+    FAIL_IF_NOT(ReadHashString(md5, string, "file", 88, 32) == 1);
+    void *ptr = ROHashLookup(hash, &md5, (uint16_t)sizeof(md5));
+    FAIL_IF_NULL(ptr);
+    PASS;
```

Comment:
I don't think that this one needs conversion, as it's a utility function for the unittest itself.

(Thinking about it, I'd say that a good way to ponder about if a function should be converted or not would be to check what the function that registers the unittests has in it...)